### DISCUSSION TITLE
Old Membership area

### DISCUSSION THREAD

**Discussion by joseph-mueller** [IviFoundation/ivifoundation.github.io#62] [General]
I originally did not include the "Membership" root page in the port because the old website basically says "do not use" at that page. Specifically it says all the membership stuff has moved to SharePoint.  

However, there are some subpages that we may want to keep.  They are: Current Members & How to Join

THere are also several pages on "Committees".  However, only 1 of the 5 described (the technical committee) still convenes (the others deferring to the BoD).   I think anything we want to say about the technical committee should be in an "about IVI" page.

So, I think the closest to the current solution would be to add some info about BoD and TC to the About page, then put Current Members and How to Join under About IVI.   

Also, we have "Corporate Contacts" which is redundant with Current Members so they should be combined.

**Comment by joseph-mueller** [comment]
If you approve of this, I can knock this out pretty quick.

**Reply by alejandro5042** [reply]
I agree, go for it.

**Reply by alejandro5042** [reply]
I like how Node.JS does their About page... they have a separate Governance page where they briefly talk about how the foundation works. Maybe we can mimic their structure? Just a thought.

**Comment by joseph-mueller** [comment]
I just noticed that the page "Corporate Contacts" is more company summary/bragging.  So the are not strictly redundant, but I would still suggest that we just combine them (simplifying maintenance also).  These blurbs need to be updated also.

**Reply by alejandro5042** [reply]
I agree. But do we even need company blurbs? Who reads those? Our actual websites will do a much better job and won't go stale. A logo (if we can get everyone to agree to that) and a link to our website should be sufficient... and in line with other foundations.

For example:

- https://openjsf.org/
- https://dotnetfoundation.org/membership/corporate-sponsorship
